Mainnet/AddressMapper/AddressMapper/AddressMapper.cs

```diff
@@ -0,0 +1,78 @@
+using System;
+using Stratis.SmartContracts;
+
+public class AddressMapper : SmartContract
+{
+    public Address GetSecondaryAddress(Address primary) => PersistentState.GetAddress($"SecondaryAddress:{primary}");
+    private void SetSecondaryAddress(Address primary, Address secondary) => PersistentState.SetAddress($"SecondaryAddress:{primary}", secondary);
+
+    private MappingInfo GetMappingInfo(Address secondary) => PersistentState.GetStruct<MappingInfo>($"MappingInfo:{secondary}");
+    private void SetMappingInfo(Address secondary, MappingInfo value) => PersistentState.SetStruct($"MappingInfo:{secondary}", value);
+    
+    private void ClearMappingInfo(Address secondary) => PersistentState.Clear($"MappingInfo:{secondary}");
+
+    public Address Owner
+    {
+        get => PersistentState.GetAddress(nameof(Owner));
+        private set => PersistentState.SetAddress(nameof(Owner), value);
+    }
```

Comment:
There probably should be a public method that allows the "Owner" to set a new `Owner` value, or possibly multiple `Approver` values. 

I see `MapAddress` allows anyone to map a secondary address to the address then sent the Transaction from, which then requires `Approve` or `Reject` by the `Owner`. One wallet or owner to handle them all might cause a backlog, all funneled through 1 person.

Reply:
`Approve` and `Reject `methods are called by address owners from Identity app. It is responsibility of the Identity app to ask user permission when a mapping requested to user's address. Identity server will handle calling Approve and Reject methods. This is not a manual process for the contract owner.  This is not going to cause backlog there. 

I added ChangeOwner method for set new Owner.